pkg/apis/vpc/v1alpha1/subnet_types.go

```diff
@@ -119,14 +120,25 @@ type SubnetAdvancedConfig struct {
 	EnableVLANExtension bool `json:"enableVLANExtension,omitempty"`
 }
 
+// Additional DHCP server config for a VPC Subnet.
+// The additional configuration must be cleared when the Subnet has DHCP relay enabled or DHCP is deactivated.
+type DHCPServerAdditionalConfig struct {
+	// Reserved IP ranges.
+	// +kubebuilder:validation::MaxItems=10
+	ReservedIPRanges []string `json:"reservedIPRanges,omitempty"`
```

Comment:
Can reservedIPRanges be updated after setting?

Reply:
Yes, it can be updated.